src/elements/content-uploader/ItemResume.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @flow
+ * @file Item resume component
+ */
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PlainButton from '../../components/plain-button/PlainButton';
```

Comment:
Minor suggestion - we often alphabetize imports in these groupings, so you could alphabetize the relative imports if you want:
- third party imports
- relative imports
- scss